clowdapp.yaml

```diff
@@ -0,0 +1,123 @@
+    - name: backend
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        minReplicas: 1
+        initContainers:
+          - command:
+            - sh
+            - -c
+              python -m manage db upgrade && python -m ros.api.main
+        resources:
+          requests:
+            cpu: ${CPU_REQUEST}
+            memory: ${MEMORY_REQUEST}
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: ${MEMORY_LIMIT}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /api/ros/v0/status
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 3
+        readinessProbe:
+          httpGet:
+            path: /api/ros/v0/status
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 3
+          periodSeconds: 10
+          successThreshold: 1
+          failureThreshold: 3
+          timeoutSeconds: 3
+        env:
+          - name: PATH_PREFIX
+            value: /api
+          - name: CLOWDER_ENABLED
+            value: ${CLOWDER_ENABLED}
+          - name: ENABLE_RBAC
+            value: "${ENABLE_RBAC}"
+    - name: processor
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        minReplicas: ${{MIN_REPLICAS}}
+        command: ["bash"]
+        args: ["-c", "python -m ros.processor.main"]
+        resources:
+          requests:
+            cpu: ${CPU_REQUEST}
+            memory: ${MEMORY_REQUEST}
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: ${MEMORY_LIMIT}
+        env:
+          - name: CLOWDER_ENABLED
+            value: ${CLOWDER_ENABLED}
+    database:
+      name: ros
+      version: 12
+    kafkaTopics:
+      - topicName: platform.inventory.events
+        partitions: 1
+      - topicName: platform.engine.results
+        partitions: 1
+
+parameters:
+- description: Enable rbac needs to be "True" or "False"
+  name: ENABLE_RBAC
+  value: "True"
+- description : ClowdEnvironment name
+  name: ENV_NAME
+  required: true
+- description: min replicas
+  name: MIN_REPLICAS
+  value: "1"
+- description: Image NAME
+  name: IMAGE
+  required: true
+  value: quay.io/cloudservices/ros-backend
+- description: Image tag
+  name: IMAGE_TAG
+  required: true
+- description: Initial cpu request.
+  displayName: CPU Request
+  name: CPU_REQUEST
+  required: true
+  value: 500m
+- description: Initial amount of memory the container will request.
+  displayName: Memory Request
+  name: MEMORY_REQUEST
+  required: true
+  value: 1Gi
+- description: Maximum amount of memory the Django container can use.
+  displayName: Memory Limit
+  name: MEMORY_LIMIT
+  required: true
+  value: 1Gi
+- description: Maximum amount of CPU the build container can use.
+  displayName: CPU Limit
+  name: CPU_LIMIT
+  required: true
+  value: '1'
+- description: Is clowder enabled
+  name: CLOWDER_ENABLED
+  value: "False"
```

Comment:
@BlakeHolifield Should this be set to `true` here?

Reply:
I'd leave it as "false" for now until you get everything migrated. We don't want to misconfigure the currently running instances.

Reply:
gotcha, thanks!